src/components/handbook-navigation/intra-page.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect } from "react";
+import { Link } from "gatsby";
+import styled from "styled-components";
+
+const JoinCommunityWrapper = styled.div`
+  .intra-page {
+    position: sticky;
+    top: 10rem;
+    right: 0rem;
+    margin-right: 1rem;
+    padding-bottom: 5rem;
+    align-items: left;
+    justify-content: space-around;
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+
+    .list {
+      color: #000000;
+    }
+    .active {
+      font-weight: 5000;
+      color: #000000;
+    }
+
+    ul {
+      list-style: none;
+      top: 3rem;
+
+      li {
+        padding-bottom: 0.5rem;
+        padding-top: 0.5rem;
+      }
+    }
+    @media only screen and (max-width: 900px) {
+      width: 0;
+      opacity: 0;
+      height: 0;
+      transition: none;
+      visibility: hidden;
+    }
+  }
+
+  .learn {
+    width: 10rem;
+  }
+`;
+
+function IntraPage({ contents }) {
+  let intapath;
+
+  useEffect(() => {
```

Comment:
@YashKamboj see useEffect hook is called after the comments are mounted into the DOM. It means we are currently setting the value of `intapath` after the components are mounted. We will need to use state in order to re-render the components after we set the value of `intapath`. You can change the code to something like this:

```
  const [intapath, setIntapath] = useState(null);

  useEffect(() => {
    const path = window.location.pathname;
    setIntapath(path);
  }, []);
```

Reply:
Thank you very much , it is working , @toth2000